public/DOM.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+const path = require('path');
```

Comment:
fs inside dom :eyes:?

Reply:
fixed!!

Reply:
thank you for your review ahmed, it really means a lot ♥